src/types/types.d.ts

```diff
@@ -13,6 +13,10 @@ export interface Config {
     rpc: string;
     deploymentBlock: number;
     watchdogTimeout?: number;
+    /**
+     * Throttle block processing to only process blocks every N blocks. Set to 1 to process every block, 2 to process every other block, etc.
```

Comment:
nit: I feel this comment is a bit redundant (it's documented in the shema above and we don't have redundant docs for any of the other parameters)

Reply:
This comment is automatically generated from the JSON schema